freshdesk_api.py

```diff
@@ -35,9 +35,13 @@ def delete(self, id):
         return self.client.req(
             requests.delete, self.api_endpoint(id), self.wrapper_name)
 
-    def get(self, id):
-        return self.client.req(
-            requests.get, self.api_endpoint(id), self.wrapper_name)
+    def get(self, id=None, args=None):
+        if args:
+            return self.client.req(
+                requests.get, self.api_endpoint(id, args), self.wrapper_name)
```

Comment:
You are breaking all other child classes (they don't accept a 2nd arg to `api_endpoint()`).